.github/workflows/samples-dart.yaml

```diff
@@ -0,0 +1,78 @@
+name: Samples Dart
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+  pull_request:
+    branches:
+      - master
+      - release/**
```

Comment:
What about using something like [5-9].[0-9].[0-9]?

ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags

Reply:
This will work for `pull_request` but for `push` it will also match tags. I guess that could be ok.

Reply:
Nvm. tags are prefixed so this should work.

Reply:
But you always use `x` for the patch version?

Reply:
yup, please use `[5-9].[0-9].x`